apiary-metastore-events/sns-metastore-events/apiary-metastore-consumers/pom.xml

```diff
@@ -15,12 +15,77 @@
         <module>privileges-grantor</module>
     </modules>
 
-    <dependencies>
-        <dependency>
-            <groupId>org.apache.hive</groupId>
-            <artifactId>hive-metastore</artifactId>
-            <version>${hive.version}</version>
-        </dependency>
-    </dependencies>
+    <properties>
+        <hive2.version>2.3.3</hive2.version>
+        <hadoop2.version>2.7.1</hadoop2.version>
+    </properties>
```

Comment:
This originally started as a WIP to unblock another internal effort and these changes went out of scope. 

The modules under `apiary-metastore-consumers` use Beeju for testing which broke w/ hive 3 as a quick replace. Some more work needs to be done w/ these tests to get this fully working, so for now I just pinned this to hive 2 to move on.

Reply:
OK, yeah, I guess we would need a version of BeeJU that also supports Hive 3. We're going to have to solve this across a wide range of projects...